flag.go

```diff
@@ -167,10 +167,13 @@ type Countable interface {
 	Count() int
 }
 
-func flagSet(name string, flags []Flag) (*flag.FlagSet, error) {
+func flagSet(name string, flags []Flag, spec separatorSpec) (*flag.FlagSet, error) {
 	set := flag.NewFlagSet(name, flag.ContinueOnError)
 
 	for _, f := range flags {
+		if c, ok := f.(customizedSeparator); ok {
+			c.WithSeparatorSpec(spec)
```

Comment:
Could this be done in apply of slice flags?

Reply:
I didn't want to change the signature of `.Apply()`

Reply:
If the flag has separator, flag.Apply() signature does not have to change

Reply:
I might've missed the memo somewhere, but how would the `Flag` get the separatorSpec

Reply:
Ahh I see the difficulty there.